app/api/v1/views/queston_views.py

```diff
@@ -101,6 +101,15 @@ def update_question(question_id):
         updated_question = question.update_question(question_id, title, description, date_posted)
         return jsonify({"Question": updated_question}),201
     abort(404)
+    
+@version1.route('/api/v1/delete_question/<int:question_id>', methods=['DELETE'])
+def delete_question(question_id):
+    """
+    Delete a question.
+    """
+    if question.delete_question(question_id):
```

Comment:
I have not been able to see where your question model is. Have you committed it to git?

Reply:
Resolved. Kindly check

Reply:
Yeah. It's okay. I would suggest on future commits you need to commit the files so that it may be easier for a reviewer to review your code.